pkg/local_object_storage/blobstor/fstree/fstree.go

```diff
@@ -304,9 +304,8 @@ func (t *FSTree) Put(prm common.PutPrm) (common.PutRes, error) {
 	if err := util.MkdirAllX(filepath.Dir(p), t.Permissions); err != nil {
 		return common.PutRes{}, fmt.Errorf("mkdirall for %q: %w", p, err)
 	}
-	if !prm.DontCompress {
-		prm.RawData = t.Compress(prm.RawData)
-	}
+	prm.RawData = t.Compress(prm.RawData)
```

Comment:
hm, a confusing change imo. suggest renaming to `CompressIfEnabled`

Reply:
That's the way it's named now, I'd not bother.

Reply:
previously it was near `if !prm.DontCompress {`, now it just always compresses

Reply:
That didn't change anything in fact, `prm.DontCompress` was an _external_ signal, while `t.Compress` can always do whatever it wants to internally.

One thing that bugs me is that our `blobstor` is not really a BLOB storage system, it handles compression at the storage level (why?), it deals with objects for whatever reason. Would be much nicer to see some `[]byte` in/out here and handle compression at engine level. But that's all a different story.